.travis.yml

```diff
@@ -4,5 +4,10 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn javadoc:jar
+  - mvn javadoc:test-aggregate
```

Comment:
Yeah, in looking at the log for the build below, I don't see any evidence that it ran tests.  Maybe I'm missing something.

https://travis-ci.org/fcrepo4-labs/migration-utils/builds/64485605